kombu/asynchronous/http/__init__.py

```diff
@@ -9,6 +9,10 @@
 
 def Client(hub: Hub | None = None, **kwargs: int) -> BaseClient:
     """Create new HTTP client."""
```

Comment:
will this work as expected? we are not using any try except block here, I'm open to better suggestions as well

Reply:
ima gonna test the code now

Reply:
the code seems to work without `pycurl` and no thrown `Exceptions`

Reply:
ok